.unreleased/pr_6811

```diff
@@ -0,0 +1 @@
+Implements: #6811 Remove deprecated timescaledb.allow_install_without_preload GUC
```

Comment:
Nit: not strictly sure it was officially deprecated. Maybe we should just say that we are removing a no longer relevant GUC?

Reply:
Good point, updated the changelog.